docs/ml-features.md

```diff
@@ -26,7 +26,9 @@ This section covers algorithms for working with features, roughly divided into t
 
 `HashingTF` is a `Transformer` which takes sets of terms and converts those sets into 
 fixed-length feature vectors.  In text processing, a "set of terms" might be a bag of words.
-The algorithm combines Term Frequency (TF) counts with the 
+A binary toggle parameter controls term frequency. When set to true all nonzero frequencies are
```

Comment:
It controls the output vector values in CountVectorizer and Term Frequency in HashingTF